developer-extension/manifest.json

```diff
@@ -9,8 +9,14 @@
     "scripts": ["background.js"],
     "persistent": true
   },
+  "content_scripts": [
+    {
+      "js": ["contentscript.js"],
+      "matches": ["<all_urls>"]
+    }
+  ],
+  "devtools_page": "devtools.html",
   "browser_action": {
-    "default_title": "Browser SDK extension",
-    "default_popup": "popup.html"
```

Comment:
Nit: When clicking on the extension icon nothing happens. It could be nice to have a tooltip telling the user where to look at or to open the devtool to the right tab.

Reply:
> open the devtool to the right tab

that could be nice but not sure that is doable

> have a tooltip telling the user where to look at

if we need to explain that, it could be done on the extension readme at some point.
I'd be in favor of removing the `browser_action` in the mean time